tests/suite_test.go

```diff
@@ -97,6 +109,15 @@ var _ = BeforeSuite(func() {
 		err = mgr.Start(ctrl.SetupSignalHandler())
 		Expect(err).ToNot(HaveOccurred())
 	}()
+
+	namespace := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: Namespace,
+		},
+	}
+	err = k8sClient.Create(context.Background(), namespace, &client.CreateOptions{})
```

Comment:
delete the resources after the running the test suite

Reply:
Okay @naveen , I will add that for namespace as well. Thanks!

Reply:
PTAL: https://github.com/red-hat-storage/mcg-osd-deployer/pull/29#discussion_r866836266

`env-test` will eventually delete all the remaining resources during the tear down of the fake cluster we test e2e features in, but I agree with Naveen that it's a good practice to delete resources as long as they are not needed anymore, just so we are sure that the flow continues in the the absence of them, and is as expected.